modules/amplify-dashboard/variables.tf

```diff
@@ -1,4 +1,4 @@
 variable "lightstep_project" {
   description = "Name of Cloud Observability project"
   type        = string
-}
+}
```

Comment:
may have lost a newline here

Reply:
This file was inconsistent with the newline at the end so I just added them across the board.